shpc/settings.yml

```diff
@@ -5,13 +5,13 @@
 # set a default module system
 # Currently lmod and tcl are supported. To request an additional system,
 # please open an issue https://github.com/singularityhub/singularity-hpc
-module_sys: lmod
+module_sys: tcl
 
 # config editor
 config_editor: vim
 
 # set a default container technology (singularity or podman)
-container_tech: singularity
+container_tech: podman
```

Comment:
You'll want to leave the defaults here as is (the container_tech and remove updated_at). Other than that, just update the version in shpc/version.py and add a corresponding note to the changelog! It's after midnight and I'm off to sleep, but I'll finish the review in the morning. Thanks for fixing this!

Reply:
My bad! the settings update is a mistake (it is too early in the morning on my side ^^).

I have fixed the commit and updated the version and changelog.

Have a good night!